tests/integration/test_oidc_client.py

```diff
@@ -0,0 +1,47 @@
+from urllib.parse import quote
+
+from flask import url_for
+import pytest
+import requests
+
+
+class TestOIDCClient(object):
+
+    def test_authenticate(self, provider, client):
```

Comment:
Need to update function to start with if_ rather than test_ as pytest.ini updated to BDD style